### PR TITLE
handle structs

### PR DESCRIPTION
**Pull request by ManApart**

### CODE REVIEW

File: test/smart_city/dataset/business_test.exs [hunk 1]
```diff
@@ -25,6 +25,12 @@ defmodule SmartCity.Dataset.BusinessTest do
       assert actual.orgTitle == "org title"
     end
 
+    test "is idempotent", %{message: biz} do
```
Comment (jeffgrunewald) [inline test/smart_city/dataset/business_test.exs]: i-DEM-potent!

File: lib/smart_city/dataset/business.ex [hunk 2]
```diff
@@ -116,7 +116,9 @@ defmodule SmartCity.Dataset.Business do
     |> elem(1)
   end
 
-  defp create(map) do
-    struct(%__MODULE__{}, map)
+  defp create(%__MODULE__{} = struct) do
```
Comment (kcarra) [inline lib/smart_city/dataset/business.ex]: I would do an `is_struct` guard here instead of forcing it to be the same struct? Maybe?

Comment (mcsearchin) [reply lib/smart_city/dataset/business.ex]: That's a hard no... *Natty*.

Comment (jeffgrunewald) [reply lib/smart_city/dataset/business.ex]: Not an option on our current version of elixir

**Pull request merged**